templates/vue/src/components/modals/NodeModal/forms/AppearanceForm.vue

```diff
@@ -181,6 +192,16 @@ export default {
       }
       return "success"
     },
+    hasPageMultiContentParent() {
+      let parentNode = this.getNode(this.getParent(this.node.id))
```

Comment:
If the node is the root node, it might not have a parent in which case I think you should test to make sure it doesn't throw any errors in the console.

Reply:
`parentNode` is undefined if no parent node is found. No errors in the console are generated in this line.